deployments/cloudfoundry/buildpack/README.md

```diff
@@ -0,0 +1,155 @@
+# OpenTelemetry Pivotal Cloud Foundry (PCF) Buildpack
+
+A [Cloud Foundry buildpack](https://docs.pivotal.io/application-service/2-11/buildpacks/) to install
+the OpenTelemetry Collector for use with PCF apps.  This will probably work for generic Cloud Foundry
+apps as well, but it is only tested and supported on Pivotal Platform.
+
+The buildpack's default functionality, as described in this document, is to deploy the OpenTelemetry Collector
+as a sidecar for the given app that's being deployed. The Collector is able to observe the app as a nozzle through
+the Loggregator Firehose. The Loggregator Firehose is one of the architectures Cloud Foundry
```

Comment:
Is it worth linking to an official doc?

Reply:
I'm sure it would be helpful! 

Is there a specific part of this paragraph that you're referring to as far as an official doc? Or should every statement have a reference?

As far as collector functionality I can link the README, but I'm not aware of any other official source of information. Please let me know if I'm missing something though.

Reply:
The Loggregator Firehose (maybe https://docs.cloudfoundry.org/loggregator/architecture.html#firehose-architecture?)

Reply:
I've added a couple links, one to the firehose and one for nozzles.